sql/mysqld.cc

```diff
@@ -5458,6 +5458,25 @@ int init_common_variables() {
                         make_query_log_name(slow_logname_path, QUERY_LOG_SLOW),
                         MYF(MY_WME)));
 
+  if (opt_general_logname != nullptr &&
```

Comment:
Shouldn't we report an error only when `opt_general_logname` is insecure AND is enabled (`--general-log=ON`)?

Reply:
Done